toolkit/tools/internal/packagerepo/repocloner/rpmrepocloner/rpmrepocloner.go

```diff
@@ -764,7 +762,7 @@ func (r *RpmRepoCloner) refreshPackagesCache() (err error) {
 
 	stdout, stderr, err := shell.Execute("tdnf", args...)
 	if err != nil {
-		logger.Log.Errorf("Failed to run 'tdnf makecache'. Stdout:\n%s\nStderr:\n%s\nError: %s.", stdout, stderr, err)
+		return fmt.Errorf("failed to run 'tdnf makecache': \n%s\n%s\n%w", stdout, stderr, err)
```

Comment:
Concatenating the stdout and stderr together without any annotation could be a little confusing. Also, I wouldn’t necessarily add the stdout and stderr to the error message without having a good idea of what the output could contain. We want the error message to be simple and readable.